src/client/service.js

```diff
@@ -0,0 +1,23 @@
+
+angular
+	.module('tl')
+	.service('tl.client.service', ['tl.service', 'tl.client.resource', function(Service, Client){
+
+		var ClientService = function(){};
+
+		/**
+		 * Generate new merchant payment token (for use with Braintree API)
+		 */
+		ClientService.prototype.paymentToken = function(success, error) {
+			return Client.paymentToken({}, success, error);
```

Comment:
should we return the ```.$promise``` here? I know we aren't very consistent on whether to return it or not, sometimes we do, sometimes we don't. We should try and standardize this.